app/ai/voice/agents/breeze_buddy/services/daily/daily.py

```
@@ -0,0 +1,119 @@
+    # Prepare runner arguments for Daily transport
+    runner_args = DailyRunnerArguments(
+        room_url=room_url,
+        token=bot_token,
+        body={
+            "lead_id": lead_id,
+            "session_id": session_id,
+        },
+    )
+
+    # Start the bot in background with the completion function
+    asyncio.create_task(daily_bot(runner_args, daily_completion_function))
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)